CONTRIBUTING.md

```diff
@@ -46,19 +46,24 @@ $ ./phpunit
 
 ### JavaScript Test Suite
 
-First, install [PhantomJS](http://phantomjs.org/) and [Google Closure
-Library](https://github.com/google/closure-library):
+First, install [PhantomJS](http://phantomjs.org/) (see the website for further 
+details or simply use your favourite package manager) and the [Google Closure
+Library](https://github.com/google/closure-library) using:
 
 ```bash
+$ cd Resources
 $ npm install google-closure-library
```

Comment:
please it as a dev dependency, so that no extra manual installation is needed.

Reply:
Good point, done in a soon-to-be pushed commit.